### PR TITLE
[MAINT] remove unused imports in examples+tools

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Comment by bashtage** [comment]
@josef-pkt Tiny PR with simple cleanups.  Should be merged.

**Comment by jbrockmendel** [comment]
@josef-pkt let's get this and similarly trivial stuff out of the way, start the second half of the week strong

**Comment by josef-pkt** [comment]
this is again fixing things in notebook exported py files, instead of syncing
e.g. http://www.statsmodels.org/devel/examples/notebooks/generated/discrete_choice_example.html

**Comment by jbrockmendel** [comment]
> this is again fixing things in notebook exported py files, instead of syncing

Updated to remove the corresponding imports in the ipynb file.

**Comment by jbrockmendel** [comment]
@josef-pkt comments have been addressed; bashtage has given go-ahead.  Any chance of action on these?

**Comment by jbrockmendel** [comment]
@josef-pkt green, no rebase needed

**Comment by jbrockmendel** [comment]
@josef-pkt getting these done will prevent them from keeping into other PRs (which cause merge conflicts which cause [...])

**Comment by bashtage** [comment]
Rebase and verify these are still relevant, and It will get in.

File: examples/notebooks/tsa_arma_0.ipynb [hunk 1]
```diff
@@ -300,7 +300,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from statsmodels.tsa.arima_process import arma_generate_sample, ArmaProcess"
```
Comment (thequackdaddy) [inline examples/notebooks/tsa_arma_0.ipynb]: It looks like you removed the world "import" here, incorrectly.

Comment (thequackdaddy) [reply examples/notebooks/tsa_arma_0.ipynb]: Causing a fail on travis docbuild.

**Comment by bashtage** [comment]
Need to get docbuild to keep travis from passing...

**Comment by thequackdaddy** [comment]
I think the recommended way was “set -e” 

That didn’t work back in the day because so much was broken/warning. Could probably retry this and be more diligent going forward?

**Pull request merged**